source/model.py

```diff
@@ -1270,11 +1270,13 @@ def __c__(self, writer):
 
 class Enumeration(Type):
 
-    def __init__(self, type_name, elems_list, enum_name = ""):
-        super(Enumeration, self).__init__(name = type_name, incomplete = False)
+    def __init__(self, elems_list, name = None):
```

Comment:
Changing arg. order is not desired. Better force a user to pass `None` for nameless enumerations explicitly.